pkg/runtime/conversion_generator.go

```diff
@@ -302,11 +302,15 @@ func (g *conversionGenerator) generateConversionsForSlice(inType, outType reflec
 }
 
 func (g *conversionGenerator) generateConversionsForStruct(inType, outType reflect.Type) error {
+	errs := []string{}
```

Comment:
Depends what you care about :)

@pweil- wants the generated fast conversions for the struct fields, which this PR accomplishes.